test/e2e/utils.go

```diff
@@ -523,6 +524,24 @@ func dhcpFlag(node string, name string) bool {
 	return gjson.ParseBytes(currentStateJSON(node)).Get(path).Bool()
 }
 
+func nodeInterfacesState(node string) []byte {
```

Comment:
Could you please add a docstring describing the returned format? `[]byte` signature is not descriptive enough